### PR TITLE
Added marginal scatter plot

### PR DESCRIPTION
**Pull request by UtkarshChemE**
I have added a plot recipe for creating marginal scatter plot. This type of scatter plot is required by some multivariate statistical analysis textbook (eg. Johnson RA, Wichern DW. Applied multivariate statistical analysis. Upper Saddle River, NJ: Prentice hall; 2002.)

### CODE REVIEW

**Comment by mkborregaard** [comment]
Could you post a picture of this? The image you're adding to the readme seems to be for marginalhist.

File: README.md [hunk 1]
```diff
@@ -126,6 +126,18 @@ iris = dataset("datasets","iris")
...
 
+## marginalscatter with DataFrames
+
+```julia
+using RDatasets
+iris = dataset("datasets","iris")
+@df iris marginalscatter(:PetalLength, :PetalWidth)
+```
+
+![marginalscatter](https://user-images.githubusercontent.com/6333339/29869938-fbe08d02-8d7c-11e7-9409-ca47ee3aaf35.png)
```
Comment (mkborregaard) [inline README.md:137]: is this the correct picture?

**Comment by UtkarshChemE** [comment]
No. I do not know where to upload the correct picture.

**Comment by mkborregaard** [comment]
Just insert it into a github comment like this one. That'll allow us to see it, and will provide you with a link you can put in the readme too.

**Comment by UtkarshChemE** [comment]
Here is the updated figure.
ms

**Pull request merged**